tests/Test/Prometheus/AbstractSummaryTest.php

```diff
@@ -66,7 +69,7 @@ public function itShouldObserveWithLabels(): void
         }
         self::assertThat(
             $this->adapter->collect(),
-            self::equalTo(
+            self::almostEqualTo(
```

Comment:
Why did you change this? By looking at commit-by-commit I can't find the specific reason. Imho it should always be equal and not almost equal

Reply:
when I was running tests on my machine, the tests were failing when comapring float numbers (on float arrays) ... the differences between values were lower that 0.00001, so I added hard rounding limit, because I think that lower differences in float numbers should be insignifficant. what do you think? did the tests work properly on your machine? I was using the dockerfile provided in this repository, so I was quite surprised that the tests were failing...

the other reason for this (IMHO the more important one) is that I changed the way of incrementing metrics values from apcu_cas to apcu_inc (in a hope, that there should be less blocking while updating metrics values). for this I need to emulate float values as integer values, because apcu_inc works with integers. since there is low probability of overflowing a 64 bit integer, I am stripping the last three digits of each integer value and reserving them for decimal digits, eg number 1,001 is represented as 1001. this conversion comes naturally with loss of precision of floating numbers, so I needed to change the tests to be almost equal and not strictly equal. what do you think about this?
btw this code is running flawlessly in our company's production systems

Reply:
if there is need for higher precision, the number of stripped digits is configurable (but globally, not per metric)

Reply:
I think we should document this (then at least in the changelog) that this is a change - IMHO this is something bc breaking-ish but I'm fine with it.